### DISCUSSION TITLE
LHstudy/comment/

### DISCUSSION THREAD

**Discussion by giscus** [LH-0124/LHstudy#3] [Announcements]
# LHstudy/comment/

https://lh-0124.github.io/LHstudy/comment/

**Comment by LH-0124** [comment]
评论功能测试

**Reply by LH-0124** [reply]
测试成功

**Comment by haoyuchen0713** [comment]
[tb20]

**Comment by IamHBW** [comment]
[【马原 wxp班】 课程体验及复习资料分享（key：马原课本pdf、腿姐背诵提纲、wxp、吴旭平）](https://www.cc98.org/topic/5925146)这个链接对应的帖子怎么消失了

**Reply by LH-0124** [reply]
忘了回复了，后面反馈了之后恢复了，不过马原现在应该已经考完了，祝你取得好成绩！

**Comment by Muchili-code** [comment]
哇，这个主页好看

**Reply by LH-0124** [reply]
gemini帮我写的[ac01]

**Comment by JerryFeatherstone** [comment]
lhgg还是太优秀啦！
